Mathlib/LinearAlgebra/Matrix/HermitianFunctionalCalculus.lean

```diff
@@ -0,0 +1,302 @@
+/-
+Copyright (c) 2024 Jon Bannon. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jon Bannon, Jireh Loreaux
+-/
+
+import Mathlib.LinearAlgebra.Matrix.Spectrum
+import Mathlib.Analysis.NormedSpace.Star.ContinuousFunctionalCalculus
+import Mathlib.Topology.ContinuousFunction.UniqueCFC
+import Mathlib.Analysis.NormedSpace.Star.Matrix
+
+/-!
+# Continuous Functional Calculus for Hermitian Matrices
+
+This file defines an instance of the continuous functional calculus for Hermitian matrices over an
+RCLike field 𝕜.
+
+## Main Results
+
+- definition of φ : the real StarAlgHom from C(spectrum ℝ A, ℝ) to (Matrix n n 𝕜) appearing in the
+                    instance.
+- instContinuousFunctionalCalculus : Instance of the Continuous functional Calculus for a hermitian
+                                     matrix A over 𝕜.
+
+## Tags
+
+spectral theorem, diagonalization theorem, continuous functional calculus
+-/
+
+section ConjugateUnits
```

Comment:
This section, and hopefully the unitary one too, should go in `Algebra.Algebra.Spectrum`, preferably even in another PR.

Reply:
I was *wondering* if you were going to suggest this. Cool!